oidc_provider/tests/test_token_endpoint.py

```diff
@@ -213,6 +213,31 @@ def test_refresh_token(self):
         response = self._post_request(post_data)
         self.assertIn('invalid_grant', response.content.decode('utf-8'))
 
+    def test_client_redirect_url(self):
+        """
+        Validate that client redirect URIs with query strings match registered
+        URIs, and that unregistered URIs are rejected.
+
+        source: https://github.com/jerrykan/django-oidc-provider/blob/2f54e537666c689dd8448f8bbc6a3a0244b01a97/oidc_provider/tests/test_token_endpoint.py
```

Comment:
That URL will be invalid as soon as the branch gets deleted. It would probably be better to either not including it, or rebase these changes against the original branch/pull request.